endpoints/k8s/k8s

```diff
@@ -543,6 +629,13 @@ function build_pod_spec() {
     echo "  ,\"kind\": \"Pod\"" >>$json
     echo "  ,\"metadata\": {" >>$json
     echo "    \"name\": \"$pod_prefix-$name\"," >>$json
+
+    if [ "$type" == "cs" ]; then
+        # server pod needs a label for loadBalancer svc to select.
+        echo "    \"labels\": {"  >>$json
+        echo "        \"app\": \"$pod_prefix-$name\""  >>$json
+        echo "    }," >>$json
+    fi
```

Comment:
Should this be always done whether the loadBalancer svc is setup/active or not?  Or should it only be done when it is active?

Reply:
To make it simpler,  just have a label always.